src/components/BlogPost/postSeparate.js

```diff
@@ -1,37 +1,69 @@
-import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-import { render } from '@testing-library/react';
-import { Container } from '@material-ui/core';
+import React from 'react';
+// import { Link } from 'react-router-dom';
+// import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import { Grid, Container } from '@material-ui/core';
 
 import blogList from './blogs';
-import blogs from './blogs.js';
+
+const styles = {
+  filteredBlogName: {
+    fontWeight: 'bold',
+    fontFamily: 'Roboto',
+    fontSize: 24,
+    marginTop: 20,
+  },
+  filteredBlogDescription: {
+    fontFamily: 'Roboto',
+    fontSize: 20,
+  },
+  sepContent: {
+    justifyContent: 'space-around',
+  },
+};
 
 function postSeparate(props) {
+  const { classes, match } = props;
+
   return (
-    <Container >
+    <Container>
       {blogList
-        .filter((blog) => blog.id == props.match.params.id)
+        .filter((blog) => blog.id === match.params.id)
         .map((filteredBlog) => {
           return (
```

Comment:
Suggestion: You could also destructure filteredBlog variable here as well but the file is smallish so it doesnt look that bad right now